app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -30,16 +30,20 @@
       <% end %>
     <% end %>
     <% if history.any? %>
-      <div class="change-notes" id="history">
-        <h3>Page history</h3>
-        <ol>
-          <% history.each do |change| %>
-            <li>
-              <time datetime="<%= change[:timestamp] %>" class="timestamp"><%= change[:display_time] %></time>
-              <%= change[:note] %>
-            </li>
-          <% end %>
-        </ol>
+      <div class="change-notes" data-module="toggle">
+        <p>
+          <a href="#full-history" data-controls="full-history" data-expanded="false">+ full page history</a>
```

Comment:
We could hide it when there is no JS, or adapt the copy so it makes a bit more sense if it's always visible.

I know it's the copy whitehall uses, but this is a good point to re-evaluate it, see it we can do something better. Eg, I suspect this doesn't sound great in a screen reader - but it would be good to test.